container/test-app/iframe/iframeContainer.html

```diff
@@ -254,6 +275,13 @@ <h3>
           Events.CHECK_PATH_EXISTS_REQUEST,
           'message received: ' + JSON.stringify(event.detail.data),
         );
+
+        if (event.callbackFn && typeof event.callbackFn === 'function') {
+          event.callbackFn({
```

Comment:
callbackFn is private and therefore should not be used, people (and AI) might look into this test app and think this is how it is supposed to be used.
Use event.callback() instead.